src/cosmos/tx/v1beta1/service.ts

```diff
@@ -74,6 +116,7 @@ export interface GetTxsEventRequest {
   events: string[];
   /** pagination defines an pagination for the request. */
   pagination?: PageRequest;
+  orderBy: OrderBy;
```

Comment:
This requires clients to set a value which does not exist in 0.42. For some reason this enum field is required by ts-proto while the message field `pagination` is optional.